kickstarts/partials/packages/includes.ks.erb

```diff
@@ -6,6 +6,7 @@ manageiq-release
 
 createrepo
 fuse-libs
+glusterfs-fuse
```

Comment:
Since this is no longer appliance specific, this should probably move to manageiq-core or manageiq-gemset rpm spec file.

Reply:
Should we add a comment that this is for "SSA on ovirt" ?

Reply:
https://github.com/ManageIQ/manageiq-rpm_build/pull/139